sdk/monitor/azingest/autorest.md

```diff
@@ -57,4 +58,10 @@ directive:
   - from: swagger-document
     where: $.paths./dataCollectionRules/{ruleId}/streams/{stream}.post
     transform: $["description"] = "Ingestion API used to directly ingest data using Data Collection Rules. Maximum size of of API call is 1 MB."
-```
\ No newline at end of file
+
+  # fix up body param to keep back-compat
+  - from: swagger-document
```

Comment:
What does this transform do? Noticed that without it the parameter will generate as type `[][]byte` instead of `[]byte`. Noticed this also happening in `azquery Table.Rows` with the newer code gen version (had to add an autorest transform). Is this supposed to be happening?

Reply:
The older code generator was accidentally treating arrays of JSON objects when `rawjson-as-bytes:true` is set as `[]byte` instead of `[][]byte`. It's a bug or a feature depending on your perspective. I assume that we wanted to keep this as `[]byte` so the transform removes the "arrayness" of the body param.